examples/README.md

```diff
@@ -422,7 +422,13 @@ cargo install wasm-bindgen-cli
 ### Build & Run
 
 Following is an example for `lighting`. For other examples, change the `lighting` in the
-following commands.
+following command.
+
+```sh
+cargo run -p build-wasm-example -- lighting
```

Comment:
I would add a not somewhere near here that you can use this to change the default example that is being built for gitpod

Reply:
to be honest I'm not completely sure how the gitpod integration will work out and would like it to be enabled before updating the "contributing" guide with this kind of info